docs/api/schema.ipynb

```diff
@@ -92,38 +143,179 @@
+      "text/plain": [
+       "['attribute(number_rows,root,1461).',\n",
+       " 'property(field,root,0).',\n",
+       " 'attribute((field,name),0,date).',\n",
+       " 'attribute((field,type),0,datetime).',\n",
+       " 'attribute((field,unique),0,1461).',\n",
+       " 'property(field,root,1).',\n",
+       " 'attribute((field,name),1,precipitation).',\n",
+       " 'attribute((field,type),1,number).',\n",
+       " 'attribute((field,unique),1,111).',\n",
+       " 'attribute((field,min),1,0).',\n",
+       " 'attribute((field,max),1,55).',\n",
+       " 'attribute((field,std),1,6).',\n",
+       " 'property(field,root,2).',\n",
+       " 'attribute((field,name),2,temp_max).',\n",
+       " 'attribute((field,type),2,number).',\n",
+       " 'attribute((field,unique),2,67).',\n",
+       " 'attribute((field,min),2,-1).',\n",
+       " 'attribute((field,max),2,35).',\n",
+       " 'attribute((field,std),2,7).',\n",
+       " 'property(field,root,3).',\n",
+       " 'attribute((field,name),3,temp_min).',\n",
+       " 'attribute((field,type),3,number).',\n",
+       " 'attribute((field,unique),3,55).',\n",
+       " 'attribute((field,min),3,-7).',\n",
+       " 'attribute((field,max),3,18).',\n",
+       " 'attribute((field,std),3,5).',\n",
+       " 'property(field,root,4).',\n",
+       " 'attribute((field,name),4,wind).',\n",
+       " 'attribute((field,type),4,number).',\n",
+       " 'attribute((field,unique),4,79).',\n",
+       " 'attribute((field,min),4,0).',\n",
+       " 'attribute((field,max),4,9).',\n",
+       " 'attribute((field,std),4,1).',\n",
+       " 'property(field,root,5).',\n",
+       " 'attribute((field,name),5,weather).',\n",
+       " 'attribute((field,type),5,string).',\n",
+       " 'attribute((field,unique),5,5).',\n",
+       " 'attribute((field,freq),5,714).']"
+      ]
      },
-     "execution_count": 4,
      "metadata": {},
-     "output_type": "execute_result"
+     "execution_count": 20
     }
    ],
    "source": [
     "dict_to_facts(schema)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This list of facts can direct be passed into Draco's constraint solver - the Clingo Solver"
```

Comment:
I think this is better documented in run.ipynb since you don't really need to run clingo here on the schema.